cylc/flow/workflow_files.py

```diff
@@ -1143,11 +1142,11 @@ def install_workflow(
             " name, using the --run-name option.")
     check_nested_run_dirs(rundir, flow_name)
     symlinks_created = {}
+    named_run = flow_name
+    if run_num:
+        named_run = os.path.join(named_run, f'run{run_num}')
```

Comment:
What about the `run_name` arg?

Reply:
Thanks, I think I have that covered now.

Reply:
If renaming as in https://github.com/cylc/cylc-flow/pull/4193/files#r631789343, I think `named_run` should be called `rel_run_dir` too

Reply:
As discussed in today's team meet, saving renaming of variables for the universal id issue/pr, so whole team can decide on agreed universal terminology.